app/public/js/profile.js

```diff
@@ -109,13 +109,10 @@ function updateRegionOptions()
 	var country_id = parseInt(getCountryElement().val());
 	var $home_region = $('#home_region');
 	$home_region.empty();
+	//Below statement enables the default region to be empty or no region
+	$home_region.append($('<option/>').attr('selected', 'selected').text("-- Select Region --"));
 	regions.forEach(function(region) {
```

Comment:
As per our gmail conversation, test the following scenarios and adjust until these work.

Sequence 1:
1. Select "Canada" and "Ontario".
1. Click "Save Profile".
1. Refresh page and "Ontario" should be selected.

Sequence 2:
1. Select "Canada" and select empty as in no province.
1. Click "Save Profile".
1. Refresh page and no province should be selected.

Sequence 3:
1. If Canada is selected, select "United States".
1. No state should be selected.